ru/news/_posts/2015-04-13-ruby-2-1-6-released.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: news_post
+title: "Вышел Ruby 2.1.6"
+author: "usa"
+translator: "gazay"
+date: 2015-04-13 12:00:00 +0000
+lang: ru
+---
+
+Вышел Ruby 2.1.6.
+
+Этот релиз содержит исправление уязвимости безопасности расширения OpenSSL.
+Детали уязвимости ниже.
+
+* [CVE-2015-1855: Ruby OpenSSL Hostname Verification](https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
+
+А также включены многие другие исправления.
+Смотри [тикеты](https://bugs.ruby-lang.org/projects/ruby-21/issues?set_filter=1&amp;status_id=5)
+и [лог изменений](http://svn.ruby-lang.org/repos/ruby/tags/v2_1_6/ChangeLog)
+за подробностями.
+
+## Скачать
+
+* [http://cache.ruby-lang.org/pub/ruby/2.1/ruby-2.1.6.tar.bz2](http://cache.ruby-lang.org/pub/ruby/2.1/ruby-2.1.6.tar.bz2)
+
+      SIZE:   12011651 bytes
+      SHA1:   380c3a5fa508fdaa2b227dbc00c56f703fd271d4
+      SHA256: 7b5233be35a4a7fbd64923e42efb70b7bebd455d9d6f9d4001b3b3a6e0aa6ce9
+      SHA512: 75d58120b5f387bcadbf6d19e85624f78c74f81b9018baef39207214673f7ebc0700ab31145acd88b4071c896ba8e1302a29c90955bcf5f8c863634125022aa6
+
+* [http://cache.ruby-lang.org/pub/ruby/2.1/ruby-2.1.6.tar.gz](http://cache.ruby-lang.org/pub/ruby/2.1/ruby-2.1.6.tar.gz)
+
+      SIZE:   15141710 bytes
+      SHA1:   426289b6647ce35ad101091825b6e7e5fce207f3
+      SHA256: 1e1362ae7427c91fa53dc9c05aee4ee200e2d7d8970a891c5bd76bee28d28be4
+      SHA512: 6563d8f39623ed5ba227725c54e630886412938bdf7c4cf03337d6c245af58d92274a098ea0e03bfd0e94970f4ee82909c366ae81db4b9317c10b92167bfc46d
+
+* [http://cache.ruby-lang.org/pub/ruby/2.1/ruby-2.1.6.tar.xz](http://cache.ruby-lang.org/pub/ruby/2.1/ruby-2.1.6.tar.xz)
+
+      SIZE:   9381724 bytes
+      SHA1:   e429644f27c243474268bf548e6fa95d05579aaf
+      SHA256: 137b27bffefd795fd97c288fff539d135f42320f8a1afddde99a34e1fbe7314e
+      SHA512: 0cf91fe7ae53a3f9c034fa5996eeed91889b942b8e595e84be4e244adc30d79aa3f540cc6f657982715069dfb14af20786557689d9a8fe4bbfc66280e84dd6cf
+
+* [http://cache.ruby-lang.org/pub/ruby/2.1/ruby-2.1.6.zip](http://cache.ruby-lang.org/pub/ruby/2.1/ruby-2.1.6.zip)
+
+      SIZE:   16671680 bytes
+      SHA1:   e79a033ab847e0d67940e31bac0debf197fad615
+      SHA256: ecbc4b97cc78e96e01375b961936133279db806044fd4d23771136dae4c1056d
+      SHA512: d1450bd013dbaabf10d7097e9dcd8c3f027110c08693ee7c94c002ea96b7e6e171c951b8b1ca3971b7f89e05b15df00ec56a006f9393889ae7f8045e9b328fad
+
+## Комментарий к релизу
+
+Спасибо всем, кто помогал с этим релизом, особенно, nagachika.
+
+Поддержка Ruby 2.1, включая этот релиз, основана на "Соглашении о стабильной версии Ruby" от [Ассоциации Ruby](http://www.ruby.or.jp/).
```

Comment:
:+1: I used https://www.transifex.com/ once, but it's not such suitable for git+md 
Any other suggestions?